resources/assets/components/SignupCard/index.js

```diff
@@ -19,12 +19,29 @@ class SignupCard extends React.Component {
 
     const signupUrl = `/signups/${signup.signup_id}`;
 
+    var campaign_run_start_date = null;
+
+    for (var key in campaign.campaign_runs.current) {
```

Comment:
let's do `for (let key in campaign.campaign_runs.current)` using `let` instead of `var` since it acts as an index that will change throughout the loop